defusedcsv/csv.py

```diff
@@ -12,37 +12,40 @@
            "unregister_dialect", "DictReader", "DictWriter", "unix_dialect"]
 
 
-def escape(payload):
+def escape(payload, assert_injection=False):
     if payload is None:
         return ''
 
     payload = str(payload)
     if payload and payload[0] in ('@', '+', '-', '=', '|', '%') and not re.match("^-?[0-9,\\.]+$", payload):
+        if assert_injection is True:
+            raise Error('The payload which shall be written is dangerous (CSV injection).')
```

Comment:
Can we define our own subclass of ``Error``, e.g. ``UnsafeError`` for this? So people can handle it specifically and show a meaningful error message to their users.

Reply:
Good point. `UnsafeError` sounds meaningful to me.